### PR TITLE
SDPA-422 HTTP publish handling gracefully non-media configuration

### PR DESCRIPTION
**Pull request by mugurrus**
transmit success on accepted HTTP code

### CODE REVIEW

File: superdesk/publish/transmitters/http_push.py [hunk 1]
```diff
@@ -42,7 +42,7 @@ def _transmit(self, queue_item, subscriber):
         if not queue_item.get(PUBLISHED_IN_PACKAGE) or not destination.get('config', {}).get('packaged', False):
             resource_url = destination.get('config', {}).get('resource_url')
             response = requests.post(resource_url, data=json.dumps(item), headers=self.headers)
-            if response.status_code != requests.codes.created:  # @UndefinedVariable
+            if response.status_code not in (requests.codes.created, requests.codes.accepted):  # @UndefinedVariable
```
Comment (vied12) [inline superdesk/publish/transmitters/http_push.py]: which variable requires `@UndefinedVariable` ?

**Comment by ioanpocol** [comment]
:+1:

**Comment by petrjasek** [comment]
:+1:

File: superdesk/publish/transmitters/http_push.py (hunk 1 above)
Comment (mugurrus) [inline superdesk/publish/transmitters/http_push.py]: .created and .accepted
see https://github.com/kennethreitz/requests/blob/master/requests/status_codes.py#L14
and https://github.com/kennethreitz/requests/blob/master/requests/status_codes.py#L85

Comment (mugurrus) [inline superdesk/publish/transmitters/http_push.py]: @UndefinedVariable is only for Eclipse to remove red underline when the code is OK but Eclipse can't figure it out.

Comment (vied12) [inline superdesk/publish/transmitters/http_push.py]: ok, thanks !

**Pull request merged**